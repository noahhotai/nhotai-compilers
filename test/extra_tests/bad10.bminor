// arrays not having size
a: array [] integer;
b: array [] array [] integer = {{1, 2}, {3, 4}};

main: function integer (z: char) = {
    c: array [] integer = {1, 2, 3, 4};
}
