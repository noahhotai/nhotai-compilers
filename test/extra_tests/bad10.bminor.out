Program parsed successfully.
Program resolved successfully.
type error: no array size given to a
type error: no array size given to b
type error: cannot assign declare array non-globally (c)
type error: no array size given to c
Program failed to typecheck.
